graphistry/client_session.py

```diff
@@ -55,9 +55,9 @@ def __init__(self) -> None:
 
         env_api_version = get_from_env("GRAPHISTRY_API_VERSION", int)
         if env_api_version is None:
-            env_api_version = 1
-        elif env_api_version not in [1, 3]:
-            raise ValueError("Expected API version to be 1, 3, instead got (likely from API_VERSION): %s" % env_api_version)
+            env_api_version = 3
+        elif env_api_version != 3:
+            raise ValueError("Expected API version to be 3 (Arrow format). Legacy API versions 1 and 2 are no longer supported. Got: %s" % env_api_version)
```

Comment:
Api equal 3. Supports arrow, parquet, CSV, json and orc via file api. Basically anything that cudf takes. I do think we do happen to do arrow uploads for DFs.

Reply:
thanks, done!